Dockerfile

```diff
@@ -1029,7 +1029,11 @@ ENV MW_AUTOUPDATE=true \
 	MW_DB_SERVER=db \
 	MW_DB_NAME=mediawiki \
 	MW_DB_USER=root \
+	MW_DB_PASS_FILE="/run/secrets/db_password /run/secrets/db_root_password" \
 	MW_DB_INSTALLDB_USER=root \
+	MW_DB_INSTALLDB_PASS_FILE=/run/secrets/db_root_password \
+	MW_ADMIN_USER_FILE=/run/secrets/mw_admin_user \
```

Comment:
is the name of the admin account really meant to be a secret? I know it was included in #192 but I don't really agree with that - was there a discussion about this that I missed?